src/clj/game/cards/events.clj

```diff
@@ -2667,6 +2667,7 @@
    :async true
    :effect (effect (make-run eid :rd nil card))
    :events [{:event :successful-run
+             :unregister-once-resolved true
```

Comment:
How is the event triggering when Maker's Eye is in the discard? Shouldn't those cards be inactive?

Reply:
It’s active in the play area and trashed the effect finishes.

Reply:
Sure, but shouldn't it be trashed as soon as the run ends and not be triggered on the second Dopplegänger run?

Reply:
No, cuz the Doppelganger run is created in 10.3.1 and the rest of the steps wait until Doppelganger's run is finished.

Reply:
Ah, so Maker's Eye isn't in the trash during the Doppelgänger run, but it's effect is read as if it said "If _that run_ is successful, access 2 additional cards from R&D". Correct?

Reply:
Yep!